cmd/logger.go

```diff
@@ -137,6 +163,26 @@ func getSource() string {
 	return fmt.Sprintf("[%s:%d:%s()]", filename, lineNum, funcName)
 }
 
+func printJSON(level logLevel, source, cause, msg string, stack []string, data ...interface{}) {
```

Comment:
pass `logrus.Level`.  logrus removal PR supposes to bring its own type, not this PR